service/aws/service.go

```diff
@@ -79,3 +82,20 @@ func ValidateAccountID(accountID string) error {
 
 	return nil
 }
+
+// GetKeyArn returns the key ARN associated with the given cluster ID.
```

Comment:
Good that this is in the AWS service. I think we'll need another resource for the KMS key since it needs to be created before cloud formation.

Reply:
Indeed, as soon as we need to create it we'll need a separate resource.